operations/helm/charts/mimir-distributed/values.yaml

```diff
@@ -384,8 +384,14 @@ rbac:
   create: true
   # For GKE/EKS/AKS use 'type: psp'. For OpenShift use 'type: scc'
   type: psp
-  # Default pod security context for mimir and cache components
-  # When using OpenShift, set
+  # -- podSecurityContext is the default pod security context for mimir and cache components
```

Comment:
```suggestion
  # -- podSecurityContext is the default pod security context for Mimir, gateway, and cache components.
```

Reply:
Also - do they have to be `null` or is omitting them sufficient?

Reply:
has to be null, custom values are merged onto chart values, so a new dict does not clear keys that are not mentioned in it